data-service/src/component-tests/java/uk/gov/gchq/palisade/component/data/common/CommonTestData.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2021 Crown Copyright
```

Comment:
This needs to be updated:
```suggestion
 * Copyright 2018-2021 Crown Copyright
```